core/src/test/java/greencity/validator/LanguageValidatorTestSerhii.java

```diff
@@ -0,0 +1,48 @@
+package greencity.validator;
+
+import greencity.service.LanguageService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.List;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class LanguageValidatorTestSerhii {
+    private final List<String> LANGUAGE_CODES = List.of("en");
+
+    @InjectMocks
+    private LanguageValidator languageValidator;
+
+    @Mock
+    private LanguageService languageService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+        when(languageService.findAllLanguageCodes()).thenReturn(LANGUAGE_CODES);
+        languageValidator.initialize(null);
+    }
+
+    @Test
+    void initialize() {
+        verify(languageService, times(1)).findAllLanguageCodes();
+    }
+
```

Comment:
Separate into two methods: one for valid input and one for invalid

Reply:
Thank you for review. Test method was separated.